modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,51 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords: upgrade, promotion, premium, button
+
+== Premum upgrade promotion option
```

Comment:
```suggestion
== Premium upgrade promotion option
```
Typo

Reply:
Corrected as part of the ‘turn that section into a partial’ push just now.